src/SUMMARY.md

```diff
@@ -10,9 +10,9 @@
 - [Uitproberen, installeren en starten](./installatie/README.md)
   - [Uitproberen](./installatie/uitproberen.md)
   - [Installeren en starten](./installatie/installeren-starten/README.md)
-    - [Methode 1: installatieprogramma downloaden en installeren (Windows)](./installatie/installeren-starten/methode-1.md)
-    - [Methode 2: installatiebestand downloaden en starten met systemd (Linux)](./installatie/installeren-starten/methode-2.md)
-    - [Methode 3: Docker Compose (Linux, macOS, Windows)](./installatie/installeren-starten/methode-3.md)
+    - [Installatieprogramma Windows](./installatie/installeren-starten/methode-1.md)
```

Comment:
Idee om de bestandnamen ook wat semantischer te maken?
- ./installatie/installeren-starten/windows.md
- ./installatie/installeren-starten/linux.md)
...

En ik denk dat de links nog korter kunnen:
- Windows
- Linux (en macOS)

(ik weet dat we macOS officieel niet ondersteunen, maar volgens mij werkt de methode meestal wel)